extensions/wikia/PaidAssetDrop/js/PaidAssetDrop.desktop.js

```diff
@@ -0,0 +1,138 @@
+define('ext.wikia.PaidAssetDrop', [
```

Comment:
I think ext.wikia.paidAssetDrop OR ext.wikia.paidAssetDrop.paidAssetDrop (first paidAssetDrop for extension name, the second for the actual module name)